frontend/src/i18n/translation.json

```diff
@@ -159,6 +176,25 @@
     "ko-KR": "전송",
     "no": "Send",
     "zh-TW": "發送",
-    "ar": "إرسال"
-  }
+    "ar": "إرسال",
+    "fr": "Envoyer"
+  },
+  "CHAT_INTERFACE$INITIAL_MESSAGE": {
+    "en": "Hi! I'm OpenDevin, an AI Software Engineer. What would you like to build with me today?",
+    "zh-CN": "Hi! I'm OpenDevin, an AI Software Engineer. What would you like to build with me today?",
```

Comment:
I do not speak any languages other than english, arabic, and french. So, I did not translate the other languages, we can also leave them blank for now since they are not really doing anything